packages/worksheet/src/config/worksheet.js

```diff
@@ -19,7 +19,61 @@ export const worksheets = [
   {
     id: 2,
     image: "",
-    heading: "Learning Made Easy",
+    heading: "Subtraction",
+    subHeading: "NCERT Workbook",
+    class: "V",
+    likes: "6",
+    comments: "0",
+    description:
+      "Worksheets help the kids in exploring multiple concepts and ideas. They develop fine motor skills and logical thinking.",
+    subject: "Math",
+    level: "Beginner",
+    grade: "VI",
+    questions: "20",
+    chapter: "2",
+    downloads: "2",
+  },
+];
+
+export const worksheetsList = [
```

Comment:
Why is this list added?

Reply:
its static data for list of worksheet now its not required so i will delete this.